rotkehlchen/inquirer.py

```diff
@@ -410,12 +410,18 @@ def _query_oracle_instances(
         ), (
             'Inquirer should never be called before the setting the oracles'
         )
-        oracles = instance._oracles
-        oracle_instances = instance._oracle_instances
-
-        if skip_onchain:
-            oracles = instance._oracles_not_onchain
-            oracle_instances = instance._oracle_instances_not_onchain
+        if from_asset.is_asset_with_oracles() is True:
```

Comment:
Hmmm why did you change this like that? Shouldn't it be the opposite?

Reply:
I changed the order of the if-else to avoid doing having to explicitly set oracle_instancee' type

Reply:
It's fine